event-exporter/sinks/stackdriver/sink_factory.go

```diff
@@ -52,6 +53,7 @@ func NewSdSinkFactory() sinks.SinkFactory {
 			"concurrent requests to Stackdriver"),
 		resourceModelVersion: fs.String("stackdriver-resource-model", "", "Stackdriver resource model "+
 			"to be used for exports"),
+		endpoint: fs.String("endpoint", defaultEndpoint, "Stackdriver base url"),
```

Comment:
Please update help string to: Base path for Stackdriver API

Reply:
Updated, CI is green. Thanks! :+1: